app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] Kindly note that the Like model belongs to the author model for the association because from the ERD the Like model was referencing the authorId for the user model and not userId. 

e.g `belongs_to :author, class_name: 'User'` 

Kindly refer back to ERD 👍🏽 
https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png?raw=true

Reply:
@vickymarz, I really appreciate your feedback, i did all the changes and learned from that. Thank you.